README.md

```diff
@@ -184,7 +184,7 @@ Global Flags:
 ```
 $ helm diff rollback -h
 
-This command compares the laset manifests details of a named release 
+This command compares the latest manifest details of a named release 
```

Comment:
It's possible that this wants a possessive, here I think we have a pair of adjectives